packages/test-utils/types/test/wrapper.ts

```diff
@@ -51,7 +51,7 @@ wrapper.findComponent(ClassComponent)
 wrapper.vm.foo
 wrapper.vm.$emit('event', 'arg')
 
-let el: HTMLElement = wrapper.element
+let el: Element = wrapper.element
```

Comment:
Can you add something like

`wrapper.find<HTMLInputElement>('#blah).element.value` 

And verify it does what it's supposed to do?

Reply:
Yes, I have added it:

```suggestion

wrapper.find<HTMLInputElement>('input').element.value
```

Check it out please